### PR TITLE
Update SDK for MangedNetwork

### PR DESCRIPTION
**Pull request by yanfa317**
This patch is to fitting modification in Swagger.
1, Add location in policy/group
2, Add tags in test case.

### CODE REVIEW

**Comment by yanfa317** [comment]
Hi, @erich-wang, can you help us review it? thanks!

File: eng/mgmt/mgmtmetadata/managednetwork.txt [hunk 1]
```diff
@@ -4,11 +4,11 @@ Commencing code generation
 Generating CSharp code
```
Comment (erich-wang) [inline eng/mgmt/mgmtmetadata/managednetwork.txt]: please follow the name pattern for this file, i.e. managednetwork_resource-manager.txt

**Review by erich-wang — CHANGES_REQUESTED** [review]

File: eng/mgmt/mgmtmetadata/managednetwork.txt (hunk 1 above)
Comment (yanfa317) [reply eng/mgmt/mgmtmetadata/managednetwork.txt]: Hi, @erich-wang , to fix it, I modify the read.md in swagger, but the new PR have not been approved yet.  To solve this issue for now, does it work if I manually rename the file?

File: sdk/managednetwork/Microsoft.Azure.Management.ManagedNetwork/src/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -8,8 +8,8 @@
 [assembly: AssemblyTitle("Microsoft Azure ManagedNetwork Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure ManagedNetwork management functions for managing the Microsoft Azure ManagedNetwork service.")]
 
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyVersion("1.0.0.1")]
+[assembly: AssemblyFileVersion("1.0.0.1")]
```
Comment (isra-fel) [inline sdk/managednetwork/Microsoft.Azure.Management.ManagedNetwork/src/Properties/AssemblyInfo.cs]: Please do not change `AssemblyVersion` and use `1.0.1.0` as `AssemblyFileVersion`.
Just FYI, `AssemblyVersion` only needs to be changed when there is a major version update.

**Review by isra-fel — COMMENTED** [review]
Hi @yanfa317 , renaming .txt file manually works for me. Just please see my inline comment to correct the version in `AssemblyInfo.cs`. Thanks

**Review by isra-fel — APPROVED** [review]

**Comment by yanfa317** [comment]
@erich-wang  I have fixed it. can you approve it? Thanks!

**Comment by isra-fel** [comment]
Erich's comment has been resolved. I'll just merge it.

**Pull request merged**